flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -1288,4 +1300,26 @@ private static CustomCommandLine loadCustomCommandLine(String className, Object.
 
         return constructor.newInstance(params);
     }
+
+    /**
+     * Get client timeout from command line via effective configuration.
+     *
+     * @param effectiveConfiguration Flink effective configuration.
+     * @return client timeout with Duration type
+     */
+    private Duration getClientTimeout(Configuration effectiveConfiguration) {
+        logAndSysout("Reading " + ClientOptions.CLIENT_TIMEOUT.key() + " from cli.");
+        return effectiveConfiguration.get(ClientOptions.CLIENT_TIMEOUT);
+    }
```

Comment:
The clientTimeout may be read from the original config. Change the log or remove it.

Reply:
@Myracle ,
Sorry that I forgot to clear the leftover logs after I refactored the code. I will do it now.